internal/config/types.go

```diff
@@ -113,7 +113,12 @@ type (
 
 	// OTel Collector Processors configuration.
 	Processors struct {
-		Batch *Batch `yaml:"-" mapstructure:"batch"`
+		Attribute *Attribute `yaml:"-" mapstructure:"attribute"`
```

Comment:
I think we want to have this more generic so that we can easily add more attribute processors in the future. Also would allow users to add their own ones as well if they want to.

Something like this:

```
Processors struct {
        Attributes []*Attribute `yaml:"-" mapstructure:"attributes"`
	Batch       *Batch     `yaml:"-" mapstructure:"batch"`
}

Attribute struct {
	key string
        action string
        value string
}
```

Reply:
Went with a slice of new type `Action` instead of a slice of `*Attributes`